vignettes/getting-started.Rmd

```diff
@@ -19,7 +19,14 @@ knitr::opts_chunk$set(
 
 This "Getting Started with rbabylon" vignette takes the user through some basic scenarios for modeling with NONMEM using `rbabylon`, introducing you to its standard workflow and functionality. 
 
-```{r results = 'hide', message=FALSE, warning=FALSE}
+```{r setup, include=FALSE}
+# NOTE: if running chunks interactively we need to load the package first
+#   because renv isolation prevents us from finding an rbabylon installation
+if (interactive()) {
+  devtools::load_all()
```

Comment:
This results in a note:

> N  checking for unstated dependencies in vignettes (1m 5s)
   '::' or ':::' import not declared from: ‘devtools’

We can [resolve](https://stackoverflow.com/a/9900160) this by adding devtools to `Suggests`.